idb/dummy.go

```diff
@@ -476,6 +476,7 @@ type TxnAssetUpdate struct {
 type AlgoUpdate struct {
 	Balance int64
 	Rewards int64
+	Closed  bool
```

Comment:
`// Closed changes the nature of the Rewards field. Balance and Rewards are normally deltas added to the microalgos and totalRewards columns, but if an account has been Closed then Rewards becomes a new value that replaces the old value (always zero by current reward logic)`

Reply:
done